packages/combobox/props.js

```diff
@@ -14,13 +14,22 @@ module.exports = {
     description:
       'A handler called when a new option is selected, takes the newly selected option value as its only parameter',
   },
-  onInputChange: {
-    type: 'function',
-    description: 'A change handler for the input',
-  },
-  onInputBlur: {
-    type: 'function',
-    description: 'An `onBlur` handler for the input',
+  inputProps: {
+    type: 'object',
+    description:
+      "Props passed to the combobox input, props listed below are not an exhaustive list, conforms to React's InputHTMLAttributes",
```

Comment:
Could we link to an exhaustive list example? If it exists.

Reply:
Yep, I thought about it, maybe https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts -- I hesitate to put a line number permalink since it could change